src/components/step/NumericCalculationStepContainer/NumericCalculationStepContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react'
+import MyDataHelps, {StepConfiguration} from '@careevolution/mydatahelps-js';
+import { calculateValue } from '../../../helpers/calculationStepFunctions/numericCalculationFunctions';
+import StepLayout from '../StepLayout'
+import { LoadingIndicator } from '../../presentational';
+
+export default function () {
+    
+    useEffect(() => {
+        MyDataHelps.getStepConfiguration().then((config: StepConfiguration) => {
+            if (!config) return; // allows test mode to work
```

Comment:
Is this needed anymore?

Reply:
Still need the config in order to get the calculation string.